pandas/core/groupby/groupby.py

```diff
@@ -3659,24 +3664,31 @@ def cumprod(
         -------
         Series or DataFrame
         """
-        nv.validate_groupby_func("cumprod", args, kwargs, ["numeric_only", "skipna"])
+        nv.validate_groupby_func("cumprod", args, kwargs, [])
```

Comment:
I think you can remove the validation here and simply remove args and kwargs since this would raise anyway now?

Reply:
So I guess we can remove `args` and `kwargs` from parameters too??